integrity-backend/input-chatbot-examples/generic-meta-content.json

```diff
@@ -0,0 +1,28 @@
+{
+  "contentMetadata":
+  {
+    "name": "Chat archive bot",
+    "mime": "application/zip",
+    "description": "Chat archive bot",
+    "author":
+    {
+      "@type": "Organization",
+      "identifier": "https://starlinglab.org",
+      "name": "Starling Lab"
+    },
+    "dateCreated": "2022-03-29T13:54:20Z",
+    "extras":
+    {
+      "channels": ["channel1","channel2"],
+      "dateRage": {
+          "from": "2022-03-29T13:00:00Z",
+          "to": "2022-03-29T14:00:00Z"
```

Comment:
Not 100% how to calculate this. Maybe first message to last message in the archive?